backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1733,3 +1733,15 @@ databaseChangeLog:
         - addNotNullConstraint:
             tableName: organization
             columnName: identity_verified
+  - changeSet:
```

Comment:
I would ditch this migration: the default value only matters if the code change hasn't rolled out yet or has been rolled back, and this migration makes the state of the app different in the "rolled back" case than in the "not yet deployed".